src/adapter/src/catalog.rs

```diff
@@ -1212,7 +1221,13 @@ impl CatalogState {
         details: EventDetails,
     ) -> Result<(), Error> {
         let user = session.map(|session| session.user().name.to_string());
-        let occurred_at = oracle_write_ts.into();
```

Comment:
@necaris Do you have an opinion on this? I was wondering the same thing but  then saw that the issue only mentioned the `mz_audit_events` table.

Reply:
That's a good question! We're definitely much more concerned about being able to test audit events -- among other things, I think there are more moving parts we need to take care of to thoroughly test storage usage -- but I think it would be useful to have this work for storage usage as well. @benesch any objections?

Reply:
I think due to the storage usage collection interval this might need to be a separate PR where a storage collection can be manually triggered after setting this timestamp, and the collection interval disabled if set or something.

Another concern: even if you set the main variable for this PR, all of the initial collections will have the system time for their creation, so you'll get a kinda wonky set of timestamps. It might be nice to also have this be a cli option or something so that the initial timestamps are also deterministic.

Reply:
> Another concern: even if you set the main variable for this PR, all of the initial collections will have the system time for their creation, so you'll get a kinda wonky set of timestamps. It might be nice to also have this be a cli option or something so that the initial timestamps are also deterministic.

You can set any system variable via the `bootstrap-system-parameter` cli option
https://github.com/MaterializeInc/materialize/blob/main/src/environmentd/src/bin/environmentd/main.rs#L430-L438

Reply:
Yeah I  think storage usage will be more complicated, so I'll go ahead and save that for a future PR should that be requested.